nuxeo-runtime/nuxeo-runtime-metrics/src/main/java/org/nuxeo/runtime/metrics/reporter/GraphiteReporter.java

```diff
@@ -0,0 +1,101 @@
+/*
+ * (C) Copyright 2020 Nuxeo (http://nuxeo.com/) and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Contributors:
+ *     bdelbosc
+ */
+package org.nuxeo.runtime.metrics.reporter;
+
+import java.net.InetSocketAddress;
+import java.util.Map;
+import java.util.Set;
+import java.util.concurrent.TimeUnit;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.nuxeo.runtime.metrics.AbstractMetricsReporter;
+import org.nuxeo.runtime.metrics.reporter.patch.NuxeoGraphiteReporter;
+
+import io.dropwizard.metrics5.MetricAttribute;
+import io.dropwizard.metrics5.MetricFilter;
+import io.dropwizard.metrics5.MetricRegistry;
+import io.dropwizard.metrics5.graphite.Graphite;
+import io.dropwizard.metrics5.graphite.GraphiteSender;
+import io.dropwizard.metrics5.graphite.GraphiteUDP;
+
+/**
+ * Patches the dropwizard metrics implementation to support metric with tags.
```

Comment:
Is it accurate? I don't see any "tags" references in the class.

Reply:
No this has been removed in another commit, the patched classes are on the patch package